src/bundle/ui-dev/src/modules/sub-items/sub.items.module.js

```diff
@@ -184,7 +185,7 @@ export default class SubItemsModule extends Component {
 
         const shouldLoadPage = !activePageItems;
 
-        if (shouldLoadPage) {
+        if (shouldLoadPage && !isLoading) {
```

Comment:
Sorry, for one more comment. :) We further discussed this solution in the frontend team and would like to adjust it slightly. Currently, the solution works without problems because during the request we are blocking in the UI any action related to the request, like: going to another page or changing the sorting order. However, if these were not blocked in the UI, the solution would not work because ongoing older request would prevent newer requests with different data. 

Because of this, we think that a better solution might be to check whether the parameters of the request - `queryConfig` - have changed in the meantime and if they did then we should proceed with the request.
This is similar to how the `useEffect` hook works (ref. https://react.dev/reference/react/useEffect#fetching-data-with-effects ) from the newer React approach (which cannot be used in this older component).
```suggestion
        if (shouldLoadPage && !this.haveRequestParamsChanged()) {
```